cloud/terraform/terraform.py

```diff
@@ -123,10 +158,10 @@ def destroy_infra(self):
 
 if __name__ == '__main__':
     tf_conf = TerraformConfigurator('aws')
-    tf_controller = TerraformerController('aws')
+    tf_controller = TerraformController('aws')
 
-    tf_conf.add_region_conf('us-east-1')
-    tf_conf.add_instance_conf('us-east-1', 'ami-0767af0854a146e3e')
+    tf_conf.configure_from_resources_json('resources.json')
```

Comment:
I would create a dir in the project called "example", "sample" or similar, where we put a real example of resources.json, so that it can be launched to locally test this Terraform controller and configurator. Then we can refer to:
```suggestion
    tf_conf.configure_from_resources_json('sample/resources.json')
```